include/boost/hana/comparing.hpp

```diff
@@ -12,6 +12,7 @@ Distributed under the Boost Software License, Version 1.0.
 
 #include <boost/hana/fwd/comparing.hpp>
 
+#include <boost/hana/detail/decay.hpp>
```

Comment:
Can you please sort the `#includes` alphabetically? This is really a small detail, but I try to keep them sorted as it makes it easier to detect duplicates and similar errors.